### PR TITLE
testing/confd: upgrade to 0.16.0

### PR DESCRIPTION
**Pull request by fchavant**
Bump to latest stable version, use included `Makefile` and add `check()` function.

### CODE REVIEW

File: testing/confd/APKBUILD [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
 # Maintainer:
 pkgname=confd
-pkgver=0.12.0_alpha3
-_ver=${pkgver/_/-}
+pkgver=0.16.0
 pkgrel=0
 pkgdesc="Manage local application configuration files using templates and data from etcd or consul"
 url="http://confd.io"
 arch="all"
 license="BSD"
```
Comment (andypost) [inline testing/confd/APKBUILD]: looks it needs update as well

File: testing/confd/APKBUILD [hunk 2]
```diff
@@ -1,37 +1,37 @@
...
-	cd "$srcdir"
-	install -Dm755 bin/confd "$pkgdir"/usr/bin/confd || return 1
+	install -Dm755 "$srcdir/src/github.com/kelseyhightower/confd/bin/confd" "$pkgdir/usr/bin/confd"
 }
 
-md5sums="1420665b612d994442e3914ab2c8f4a7  confd-0.12.0-alpha3.tar.gz"
-sha256sums="b679b35afb593bd9472cbf35c778829288c85c7b92678747d3a059ea0d74c16c  confd-0.12.0-alpha3.tar.gz"
-sha512sums="4feb81952a5cd2fcb8250cf5df93b6ceeb9a2e5173f45f7c5cbdcd2de59acf7d85a60c1513520c601e7b7a1257d26c912877eaac53aafe9e4e278e4e9cc6ab15  confd-0.12.0-alpha3.tar.gz"
+md5sums="9678e35cdb7899549a3a88116e7ab74f  confd-0.16.0.tar.gz"
+sha256sums="4a6c4d87fab77aa9827370541024a365aa6b4c8c25a3a9cab52f95ba6b9a97ea  confd-0.16.0.tar.gz"
```
Comment (andypost) [inline testing/confd/APKBUILD]: md5 & sha256 better to remove

File: testing/confd/APKBUILD (hunk 1 above)
Comment (andypost) [reply testing/confd/APKBUILD]: Actually it is MIT https://github.com/kelseyhightower/confd/blob/master/LICENSE
The list is https://spdx.org/licenses/

**Comment by fchavant** [comment]
Removed md5 and sha256 checksums, updated license and changed commit message as requested.

**Pull request closed without merging**